releasenotes/notes/resonator-spectroscopy-89f790412838ba5b.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    This change introduces a new experiment that allows users to run spectroscopy
+    on readout resonators. This is done by attaching a custom pulse-schedule to
+    the measure instruction. Furthermore, a new data processing node is introduced
+    that takes the absolute value of the IQ points. This node is needed to analyse
+    readout resonator spectroscopy IQ data since it rotates around in the IQ plane.
```

Comment:
```suggestion
  - |
    :py:class:`~qiskit_experiments.library.ResonatorSpectroscopy` has been added. This experiment performs spectroscopy experiment on a measurement channel. This is done by attaching a custom pulse-schedule to
    the measure instruction.
  -|
    New data processing node :py:class`~qiskit_experiments.data_processing.nodes.ToAbs` has been added.
    This takes the absolute value of the IQ points. This node may be needed to analyze
    readout resonator spectroscopy IQ data since it rotates around in the IQ plane. 
```

Reply:
See here: 5a07e94